addons/l10n_in_withholding/models/account_move.py

```diff
@@ -34,6 +34,7 @@ class AccountMove(models.Model):
     )
     l10n_in_tcs_tds_warning = fields.Char('TDC/TCS Warning', compute="_compute_l10n_in_tcs_tds_warning")
     l10n_in_display_higher_tcs_button = fields.Boolean(string="Display higher TCS button", compute="_compute_l10n_in_display_higher_tcs_button")
+    l10n_in_applicable_section_ids = fields.Many2many('l10n_in.section.alert', string="TDS/TCS Applicable Sections", compute="_compute_l10n_in_tcs_tds_warning")
```

Comment:
Don't add fields just for view purposes.
find an alternative like showing `account.move.line` with the group by `l10n_in_tds_tcs_section_id` and `move_id`